### PR TITLE
Minecard

### PR DESCRIPTION
**Pull request by harsh-quadB**
## Description
Please include a summary of the changes and the purpose of this PR.

- What does this PR do?
- Why is it needed?

## Screenshots
Include screenshots for **both desktop and mobile** versions.

### Desktop:
[Add screenshot here]

### Mobile:
[Add screenshot here]

## Approach
Briefly explain the approach taken to implement the changes. 

- How did you solve the problem?
- Any particular design patterns or optimizations?

## Issue Number (if any)
Link to the issue this PR addresses, if applicable:

Fixes #[issue_number]

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/game_frontend/src/components/ui/Minecard/index.tsx
Comment (BeginArjun) [inline src/game_frontend/src/components/ui/Minecard/index.tsx:5]: All interfaces should be stored inside `types/[Component]/index.d.ts`

File: src/game_frontend/src/components/ui/Minecard/index.tsx
Comment (BeginArjun) [inline src/game_frontend/src/components/ui/Minecard/index.tsx:122]: No need for this component. Pls remove it

File: src/game_frontend/src/components/ui/Minecard/index.tsx
Comment (BeginArjun) [inline src/game_frontend/src/components/ui/Minecard/index.tsx:56]: This should be the default export

File: src/game_frontend/src/components/ui/Minecard/index.tsx
Comment (BeginArjun) [inline src/game_frontend/src/components/ui/Minecard/index.tsx:25]: Static data shouldn't be in this file. Kindly remove it or comment it

File: src/game_frontend/src/components/ui/Minecard/index.tsx
Comment (BeginArjun) [inline src/game_frontend/src/components/ui/Minecard/index.tsx:19]: Not specific to this component

File: src/game_frontend/src/components/ui/Minecard/index.tsx
Comment (BeginArjun) [inline src/game_frontend/src/components/ui/Minecard/index.tsx:72]: Why is a default image being used? This shouldn't be here.

File: src/game_frontend/src/components/ui/Minecard/index.tsx
Comment (BeginArjun) [inline src/game_frontend/src/components/ui/Minecard/index.tsx:110]: Where is `ic` written in Figma?

File: src/game_frontend/src/components/ui/Minecard/index.tsx
Comment (BeginArjun) [inline src/game_frontend/src/components/ui/Minecard/index.tsx:107]: It should be
```jsx
<span className="mine-card-level">Lvl. {level}</span>
```

File: src/game_frontend/src/components/ui/Minecard/index.tsx
Comment (BeginArjun) [inline src/game_frontend/src/components/ui/Minecard/index.tsx:61]: default level should be 0

**Review by BeginArjun — CHANGES_REQUESTED** [review]
Kindly first pull Changes from the PR #19 after it gets approved and after then start working on these changes.

**Review by ArjunQBTech — APPROVED** [review]
Great Work!

**Pull request merged**
